src/redux/reducer/DataHub.js

```diff
@@ -0,0 +1,27 @@
+import {ActionType} from '../action/ActionType';
+const InitialState = {
+  loginpage: {
+    Email: '',
+    Password: '',
+  },
+  RegisterPage: {
+    name: '',
+    registeremail: '',
+    cellNumber: '',
+    password: '',
+  },
+};
+const userReducer = (state = InitialState, action) => {
+  switch (action.type) {
+    case ActionType.LOGIN_USER:
+      // console.log(loginpage);
+      return {...state, loginpage: {...action.payload}};
+
+    case ActionType.REGISTER_USER:
+      return {...state, RegisterPage: {...action.payload}};
+    default:
+      return state;
+  }
+};
+
+export default userReducer;
```

Comment:
Why the file name datahub when reducer's name is userReducer?

Reply:
Done